pkg/crc/machine/machine.go

```diff
@@ -769,12 +773,7 @@ func updateSSHKeyPair(sshRunner *crcssh.SSHRunner) error {
 	return err
 }
 
-func configureCluster(ocConfig oc.OcConfig, sshRunner *crcssh.SSHRunner, proxyConfig *network.ProxyConfig, pullSecret, instanceIP string) (rerr error) {
-	sd := systemd.NewInstanceSystemdCommander(sshRunner)
-	if _, err := sd.Start("kubelet"); err != nil {
-		return fmt.Errorf("Error starting kubelet: %s", err)
-	}
-
+func configureCluster(ocConfig oc.OcConfig, sshRunner *crcssh.SSHRunner, proxyConfig *network.ProxyConfig, pullSecret, instanceIP string, sd *systemd.InstanceSystemdCommander) (rerr error) {
```

Comment:
`configProxyForCluster` can recreate a `systemd.Instance` from the `sshRunner` when needed, I don't think it's really useful to pass around `sd` in function args.